families/supplychain/python/sawtooth_supplychain/client.py

```diff
@@ -195,9 +194,9 @@ def application_accept(self, record_identifier,
             SupplyChainPayload.APPLICATION_ACCEPT,
             ApplicationAcceptPayload(
                 record_identifier=record_identifier,
-                applicant=applicant,
+                applicant=applicant_public_key,
                 type=application_type,
-                timestamp=timestamp or time.time()),
+                timestamp=timestamp or int(time.time())),
```

Comment:
There is no discussion the spec as to what granularity this timestamp should be.  Is it millis or seconds?  If the former you should probably multiply this by 1000 before calling `int`